homeassistant/bootstrap.py

```diff
@@ -262,8 +290,7 @@ def enable_logging(hass, verbose=False, daemon=False, log_rotate_days=None):
                 }
             ))
         except ImportError:
-            _LOGGER.warning(
-                "Colorlog package not found, console coloring disabled")
```

Comment:
Because it is not a warning, it's a feature that will not be enabled. We should document it somewhere, or maybe decide to install the package by default.